### PR TITLE
Integration test

### LINKED ISSUE #66: Create an end to end user flow test

### PR DESCRIPTION
**Pull request by camillelnne**
Fixes #66.

This PR introduces our first end-to-end test !
The test behaves like a new user: logins, create an account, explore the different pages and finally create a new post and refresh to see it.

In more details:
- Begins in log in page
- Create a new account by filling the fields and confirms
- Checks the user profile: posts and comments
- Navigate from home to all pages using the bottom bar
- Creates a new post by clicking on the button of the bottom bar and submit it
- Refresh the page and verify that the new post is displayed

For now, the test is run locally with ` flutter test integration_test/app_test.dart` in the command line, to avoid overhead in the CI.

### CODE REVIEW

File: integration_test/app_test.dart [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+  // Map
+  await tester.tap(find.text("Map"));
+  await tester.pumpAndSettle();
+  expect(find.text("Proxima"), findsOneWidget);
+
+  // Home (Feed)
+  await tester.tap(find.text("Feed"));
+  await tester.pumpAndSettle();
+  expect(find.byType(HomePage), findsOneWidget);
+}
```
Comment (Aderfish) [inline integration_test/app_test.dart:168]: I will add a test for the new post navigation for completeness.

Comment (camillelnne) [reply integration_test/app_test.dart:175]: Great catch, thanks!

**Review by Aderfish — APPROVED** [review]
Looks good to me, I just added a small extra step in the bottom navigation. Well done writing this first integration test on a full user flow.

**Review by yoannLafore — APPROVED** [review]
Great work :tada: 
The end to end user flow looks to be robuslty tested.
The PR looks good to me :+1:

**Pull request merged**